content/en/blog/2022/enhancements-process-update.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: blog
+title: "Enhancements Opt-in Process Change for v1.26"
+date: 2022-09-06
+slug: enhancements-opt-in
+---
+
+# Enhancements Opt-in Process Change for v1.26 
+
+## Context and Motivations
+Since the inception of the Kubernetes release team, we have used a spreadsheet to keep track of enhancements for the release. The project has scaled massively in the past few years, with almost a hundred enhancements collected for the 1.24 release. This process has become error-prone and time consuming. A lot of manual work is required from the release team and the SIG leads to populate KEPs data in the sheet. We have received continuous feedback from our contributors to streamline the process.
```

Comment:
nit: newline for consistency with other sections
```suggestion

Since the inception of the Kubernetes release team, we have used a spreadsheet to keep track of enhancements for the release. The project has scaled massively in the past few years, with almost a hundred enhancements collected for the 1.24 release. This process has become error-prone and time consuming. A lot of manual work is required from the release team and the SIG leads to populate KEPs data in the sheet. We have received continuous feedback from our contributors to streamline the process.
```